drivers/entropy/Kconfig

```diff
@@ -6,19 +6,27 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+config ENTROPY_MUST_USE_OUT_OF_TREE_DRIVER
+  bool
+  help
+	  This option can be enabled by an out-of-tree library if it
+	  needs to take control over the entropy driver.
+
 menuconfig ENTROPY_GENERATOR
 	bool "Entropy Drivers"
 	help
 	  Include entropy drivers in system config.
 
-if ENTROPY_GENERATOR
 
+if ENTROPY_GENERATOR
```

Comment:
Bit hard to read. Looks like the `if ENTROPY_GENERATOR` goes just with the `source`s, due to the scrunching up.

Reply:
You want to add a few more blank lines then? Around the two if's and the endif?

Reply:
Thinking it could be e.g.

```
if X

if Y
blah
blah
endif # Y

endif # X
```

instead of

```
if X
if Y
blah
blah
endif # Y

endif # X
```

Second one kinda makes it look like the `if X` only attaches to the first block.